src/components/publications_list/PublicationAddEdit.tsx

```diff
@@ -29,118 +19,335 @@ interface PublicationAddEditProps {
+  publishedDate?: ValidationError
+  authors?: ValidationError
   bibliographicCitation?: ValidationError
+  datasetCitation?: ValidationError
+  journal?: ValidationError
+  doi?: ValidationError
+  url?: ValidationError
+  access?: ValidationError
 }
+
+const defaultPublication: Publication = {
+  title: '',
+  publishedDate: '',
+  authors: [],
+  pubmedId: '',
+  bibliographicCitation: '',
+  datasetCitation: '',
+  citation: false,
+  publicationId: '',
+  studyId: '',
+  journal: '',
+  doi: '',
+  url: '',
+  access: '',
+  tags: [],
+}
+
 export default function PublicationAddEdit(props: PublicationAddEditProps): React.JSX.Element {
   const { id, publication, publications, closeAction, onPublicationChange } = props
+  const initialPublication = publication || defaultPublication
+
+  // Ensure at least one author row is present and bound to state
+  const [newPublication, setNewPublication] = useState<Publication>({
+    ...initialPublication,
+    authors:
+            Array.isArray(initialPublication.authors) && initialPublication.authors.length > 0
+              ? initialPublication.authors
+              : [{ name: '', orcId: '' }],
+  })
 
-  const [newPublication, setNewPublication] = useState<Publication>(publication || defaultPublication)
   const [validation, setValidation] = useState<Validation>({})
+  const [submitted, setSubmitted] = useState<boolean>(false)
+  const [touched, setTouched] = useState<Record<string, boolean>>({})
+  const [tagsInput, setTagsInput] = useState<string>((publication?.tags ?? []).join(', '))
+
+  const applyValidation = (draft: Publication, full: boolean) => {
+    const all = calcPublicationErrors(draft)
+    if (full) {
+      setValidation(all)
+      return
+    }
+    const filtered: Validation = {}
+    for (const [k, v] of Object.entries(all)) {
+      if (touched[k]) filtered[k as keyof Validation] = v
+    }
+    setValidation(filtered)
+  }
+
+  const updatePublication = (updated: Publication) => {
+    setNewPublication(updated)
+    if (submitted) applyValidation(updated, false)
+  }
+
+  const markTouched = (key: string) => {
+    setTouched(prev => (prev[key] ? prev : { ...prev, [key]: true }))
+  }
 
   const onChange = ({ key, value }: FormFieldChange) => {
-    const publicationToSet: Publication = { ...newPublication, [key]: value }
-    setNewPublication(publicationToSet)
-    setValidation(calcPublicationErrors(publicationToSet))
+    markTouched(key)
+    if (key === 'tags') {
+      const text = String(value)
+      setTagsInput(text)
+      updatePublication({
+        ...newPublication,
+        tags: text.split(',').map(t => t.trim()).filter(Boolean),
+      })
+    }
+    else {
+      updatePublication({ ...newPublication, [key]: value })
+    }
+  }
+
+  // Enable Add only when all current authors have name + valid ORCID
+  const disableAddAuthor = newPublication.authors.some(
+    a => a.name.trim() === '' || a.orcId.trim() === '' || !ORCID_REGEX.test(a.orcId),
+  )
```

Comment:
If we want ORCID to be optional as I suspect, I think we'd want something closer to the following (which would also be worth a quick local UI check):

```suggestion
  // Enable Add only when all current authors have name + valid ORCID
  const disableAddAuthor = newPublication.authors.some(
    a => a.name.trim() === '' || (a.orcId.trim() !== '' && !ORCID_REGEX.test(a.orcId)),
  )
```